tutorials/34_Model_Based_Evaluation_of_RAG_Pipelines.ipynb

```diff
@@ -0,0 +1,4480 @@
+{
```

Comment:
<ul><li>There's a typo in "...Haystack's integrated <strong>evaluation</strong> frameworks"</li><li>Can you very briefly explain what "Context Relevance" and "Critique language" metrics are? Links would work as well</li><li>We can now replace the UpTrainEvaluator PR link with docs link: <a href="https://docs.haystack.deepset.ai/v2.0/docs/uptrainevaluator" target="_blank">https://docs.haystack.deepset.ai/v2.0/docs/uptrainevaluator</a></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
done